src/renderer/services/slippi/graphqlEndpoints.ts

```diff
@@ -38,7 +39,7 @@ export const QUERY_VALIDATE_USER_ID: TypedDocumentNode<
 
 export const QUERY_GET_USER_KEY: TypedDocumentNode<
```

Comment:
This should be renamed since we're now fetching more than just the key and potentially anything related to the user.

```suggestion
export const QUERY_GET_USER_DATA: TypedDocumentNode<
```